### ISSUE TITLE
TGBot

### ISSUE THREAD

**Issue by piorphism** [openprf/betbot#1]
do u have a telegram bot that working as this betbot? if u have pls provide with the link

**Comment by AfanasyevDataArt** [comment]
No, i don't. I didn't try to run betbot code since telegram had been banned in Russia. It was educational project for me. I don't have capacity to implement ban bypass now. But you can run it by your self.
